### ISSUE TITLE
Enabling addon in cppcheck doesn't show any diagnostics

### ISSUE THREAD

**Issue by befedo** [mfussenegger/nvim-lint#814]
Hi there,

I've configured `nvim-lint` with lazy.nvim like this:
```{lua}
  {
    "mfussenegger/nvim-lint",
    config = function ()
      local lint = require("lint")

      lint.linters_by_ft = {
        c = { "cppcheck" },
        cpp = { "cppcheck" },
      }

      lint.linters.cppcheck.args = {
        "--addon=y2038",
        unpack(lint.linters.cppcheck.args)
      }

      vim.api.nvim_create_autocmd({ "BufEnter", "BufWritePost" }, {
        callback = function()
          require("lint").try_lint()
        end,
      })
    end,
  }
```
When uncommenting the y2038 addon diagnostics are visible again. I don't see any information with `:messages` and cppcheck runs in both ways. If I can help debug this on my side, please let me know.
